frontend/src/components/Order/components/PostcardAndComment/PostcardAndComment.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import styles from './PostcardAndComment.module.scss';
+import Button from '../../../ui-kit/components/Button';
+
+const PostcardAndComment = ({ postcardData, setDataOnSubmit }) => {
+  const [showPostcardText, setShowPostcardText] = useState(
+    postcardData?.postcard ? postcardData.postcard === 'Add postcard' : true,
+  );
+  const [showComment, setShowComment] = useState(postcardData?.comment !== '');
+
+  const { register, handleSubmit } = useForm({
+    defaultValues: {
+      postcard: postcardData?.postcard ? postcardData.postcard : 'Add postcard',
```

Comment:
Here you can simply use postcardData?.postcard || 'Add postcard'

Reply:
Done.